include/realizations/catchment/Formulation_Manager.hpp

```diff
@@ -285,17 +289,20 @@ namespace realization {
              * 
              * @return std::string of the output root directory
              */
-            std::string get_output_root() const noexcept {
+            std::string get_output_root() const {
                 const auto output_root = this->tree.get_optional<std::string>("output_root");
-                if (output_root != boost::none && *output_root != "") {
-                    // Check if the path ends with a trailing slash,
-                    // otherwise add it.
-                    return output_root->back() == '/'
-                           ? *output_root
-                           : *output_root + "/";
```

Comment:
For reference, this manipulation was added because the output file paths are constructed by concatenation, i.e.

https://github.com/NOAA-OWP/ngen/blob/0917f9b021e081f01d42966ad92ee6370f3eca54/src/core/HY_Features.cpp#L35

https://github.com/NOAA-OWP/ngen/blob/0917f9b021e081f01d42966ad92ee6370f3eca54/src/NGen.cpp#L443

Without checking/adding the trailing slash, or modifying the code at the call sites, we may have paths of the form `/some/output/dirnex-01_output.csv` instead of `/some/output/dir/nex-01_output.csv` for example.